src/main/java/seedu/guest/logic/commands/ResetRoomCleanCommand.java

```diff
@@ -0,0 +1,66 @@
+package seedu.guest.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.guest.logic.commands.exceptions.CommandException;
+import seedu.guest.model.Model;
+import seedu.guest.model.guest.Bill;
+import seedu.guest.model.guest.DateRange;
+import seedu.guest.model.guest.Email;
+import seedu.guest.model.guest.Guest;
+import seedu.guest.model.guest.IsRoomClean;
+import seedu.guest.model.guest.Name;
+import seedu.guest.model.guest.NumberOfGuests;
+import seedu.guest.model.guest.Phone;
+
+
+/**
+ * Changes isRoomClean field of all guest in current list to "no".
+ */
+public class ResetRoomCleanCommand extends Command {
+
+    public static final String COMMAND_WORD = "resetRoomClean";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Changes all guest RoomClean status in current list to \"no\".\n"
+            + "Example: " + COMMAND_WORD;
+
+    public static final String MESSAGE_SUCCESS = "Reset all guests' Room Clean status in current list to \"no\".";
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        requireNonNull(model);
+        List<Guest> lastShownList = model.getFilteredGuestList();
+
+        Guest guestToEdit;
+        Guest editedGuest;
+
+        for (int index = 0; index < lastShownList.size(); index++) {
```

Comment:
Does this only reset the lastShownList? (ie not the entire list but only the list that is searched for) (eg. Find Alex -> resetRoomClean)

Reply:
Yes only last shown list. I thought I should standard how edits are made so that it's more intuitive for users. Currently, delete and edit commands also only work on last shown list. But I also understand that it make sense to change all the guests at once.

Reply:
I have discussed with Justin and will make the necessary edits to edit the whole list instead! Thanks for bringing it up!